azure/services/agentpools/spec.go

```diff
@@ -269,24 +269,18 @@ func (s *AgentPoolSpec) Parameters(ctx context.Context, existing interface{}) (p
 			log.V(4).Info("no changes found between user-updated spec and existing spec")
 			return nil, nil
 		}
-		log.V(4).Info("found a diff between the desired spec and the existing agentpool", "difference", diff)
```

Comment:
Was changing the verbosity here intentional? I don't have an opinion on what it should be, but this is the kind of thing I would definitely do to help debug and then forget to change back 😄

Reply:
Yep, forgot to revert a debugging change. :-) Will fix.

Reply:
Fixed.